.travis.yml

```diff
@@ -42,8 +42,7 @@ script:
       cargo test --verbose;
     fi
   - if [ "$CLIPPY" ]; then
-      cargo install cargo-update;
-      cargo install-update -i cargo-update clippy;
+      cargo install -qf clippy &&
```

Comment:
Also:
  * why `-q`, and
  * why `&&`?

Reply:
`-q` because it was noisy, and `&&` because I didn't want it to continue trying stuff after one of the commands failed.

Reply:
Noise doesn't matter v0v